filesystems/xfs/xfstests/README.md

```diff
@@ -0,0 +1,28 @@
+xfstests test suite
+=====================================
+Wrapper for xfstests test suite, see http://git.kernel.org/?p=fs/xfs/xfstests-dev.git;a=summary \
+Test Maintainer: [Xiong Murphy Zhou](mailto:xzhou@redhat.com)
+
+How to run it
+-------------
+Make sure you have at least 25G free disk space in this partition
```

Comment:
Would be nice to mention why you'd need 25GB.

Reply:
I think @jencce could answer this question better

Reply:
That's what we need for best coverage in beaker tests. It's not mandatory. We can remove it.

Reply:
I removed this